homeassistant/components/tuya/select.py

```diff
@@ -128,6 +128,40 @@
             translation_key="motion_sensitivity",
         ),
     ),
+    # Smart Camera - Low power consumption camera
+    # Undocumented, see https://github.com/home-assistant/core/issues/132844
+    "dghsxj": (
```

Comment:
This is a full copy & paste of the `sp` product category?

Why didn't we alias it instead of duplicating the full set of descriptions?